src/elements/content-preview/preview-header/PreviewHeader.scss

```diff
@@ -45,6 +45,7 @@
     .ba-point-post-cancel-container,
     .ba-draw-post-cancel-container {
         position: absolute;
+        top: 6px;
```

Comment:
Will this affect the other browsers? Why the misalignment only happened in IE11, not the other browsers?

Reply:
Not sure, seems like the other browsers just handle it more gracefully